cabal.project

```diff
@@ -1,8 +1 @@
 packages: .
-
-constraints: hedgehog-classes -aeson -semirings -comonad -vector -primitive
```

Comment:
Let's keep this line with constraints. It disables some heavy dependencies and makes the course compilation much faster for the first time  🚀